src/meshes/t8code_mesh.jl

```diff
@@ -100,6 +100,125 @@ function Base.show(io::IO, ::MIME"text/plain", mesh::T8codeMesh)
     end
 end
 
+"""
+    T8codeMesh{NDIMS, RealT}(forest, boundary_names; polydeg = 1, mapping=identity)
+
+Create a 'T8codeMesh'.
```

Comment:
Maybe expand a litte on when it's useful to use this ctor and not the other one? E.g., including an explanation of what a "committed forest" is or what the typical use case is you have in mind for this ctor?

Reply:
Done. Removed `commited` since this technical detail should not be relevant for most users.